_includes/releases/v22.1/v22.1.0-alpha.2.md

```diff
@@ -0,0 +1,640 @@
+- When the `sql.telemetry.query_sampling.enabled` cluster setting is enabled, SQL names and client IPs are no longer redacted in telemetry logs. [#76676][#76676]
+
+<h3 id="v22-1-0-alpha-2-general-changes">General changes</h3>
+
+- The following metrics were added for observability of cancellation requests made using the PostgreSQL wire protocol:
+    - `sql.pgwire_cancel.total`
+    - `sql.pgwire_cancel.ignored`
+    - `sql.pgwire_cancel.successful`
+
+    The metrics are all counters. The `ignored` counter is incremented if a cancel request was ignored due to exceeding the per-node rate limit of cancel requests. [#76457][#76457]
+- Documentation was added describing how jobs and scheduled jobs functions and are used in CockroachDB [#73995][#73995]
+
+<h3 id="v22-1-0-alpha-2-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Client certificates may now be provided for the `webhook` [changefeed sink](../v22.1/changefeed-sinks.html). [#74645][#74645]
+- CockroachDB now redacts more potentially sensitive URI elements from changefeed job descriptions. This is a breaking change for workflows that copy URIs. As an alternative, the unredacted URI may be accessed from the jobs table directly. [#75174][#75174]
+- Changefeeds now outputs the topic names created by the Kafka sink. Furthermore, these topic names will be displayed in the [`SHOW CHANGEFEED JOBS`](../v22.1/show-jobs.html#show-changefeed-jobs) query. [#75223][#75223]
+- [Backup and restore](../v22.1/take-full-and-incremental-backups.html) jobs now allow encryption/decryption with GCS KMS [#75750][#75750]
+- [Kafka sinks](../v22.1/changefeed-sinks.html#kafka) support larger messages, up to 2GB in size. [#76265][#76265]
+- Added support for a new SQL statement called `ALTER CHANGEFEED`, which allows users to add/drop targets for an existing changefeed. The syntax of the statement is: `ALTER CHANGEFEED <job_id> \{\{ADD|DROP\} <targets>\}...`
```

Comment:
```suggestion
- Added support for a new SQL statement called `ALTER CHANGEFEED`, which allows users to add/drop targets for an existing changefeed. The syntax of the statement is: `ALTER CHANGEFEED <job_id> {{ADD|DROP} <targets>}...`
```

Reply:
The backslash escapes are rendered in the HTML.

Reply:
Dang. The build thinks it's Liquid, and breaks. Going to have to figure out something clever here.

Reply:
Ok, I can use `{% raw %}` and `{% endraw %}` to escape this.